jmclient/test/test_boltzmann.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
```

Comment:
I think `from builtins import *` should probably go here to force python 3 behavior